audioscrape/__main__.py

```diff
@@ -2,14 +2,20 @@
 """Download audio."""
 import argparse
 import sys
+import os
 
 from . import soundcloud, youtube
 
 
-def download(query, include=None, exclude=None, quiet=False, overwrite=False):
+def download(query, include=None, exclude=None, quiet=False,
+             overwrite=False, fileformat=None):
     """Scrape various websites for audio."""
-    youtube.scrape(query, include, exclude, quiet, overwrite)
-    soundcloud.scrape(query, include, exclude, quiet, overwrite)
+    # create subdirectory for converted audio files if --fileformat tag set
```

Comment:
Minor nitpick but capitalize and have normal sentences with periods for code comments, please. 🐼

Reply:
This is your show, so no problem there! Will do 👍